LayoutTests/svg/dynamic-updates/SVGGraphicsElement-transform-translate-single-parameter.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML//EN">
+<html>
+<head>
+<script src="resources/SVGTestCase.js"></script>
+<script src="../../resources/js-test-pre.js"></script>
```

Comment:
`js-test.js` is recommend for new tests instead of `-pre.js` and `post.js` combo or singular use.

Reply:
Or even better, make a WPT test.